connection/endpoint.go

```diff
@@ -1,18 +1,20 @@
 package connection
 
 import (
-	"os"
-
-	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
 )
 
-const AWS_ENDPOINT = "AWS_ENDPOINT"
-
-func SetEndPoint(config aws.Config) aws.Config {
-	newConfig := config
-	awsEndpoint := os.Getenv(AWS_ENDPOINT)
-	if awsEndpoint != "" {
-		newConfig.Endpoint = aws.String(awsEndpoint)
-	}
-	return newConfig
+func SetEndPoint(endpoint string) config.LoadOptionsFunc {
```

Comment:
```suggestion
func EndpointOption(endpoint string) config.LoadOptionsFunc {
```

Since its no longer setting anything

Also the package connection is a bit iffy on this one, maybe config, where other config related functionality could reside in the future? Nitpick

Reply:
thanks for the review Nikola!